src/components/StakingVaults/Cells.tsx

```diff
@@ -26,9 +28,14 @@ type AssetCellProps = {
   postFix?: string
   showTeaser?: boolean
   showAssetSymbol?: boolean
+  icons?: Array<string>
 }
 
 const buildRowTitle = (asset: Asset, postFix?: string, showAssetSymbol?: boolean): string => {
+  if (asset.assetId === foxEthLpAssetId) {
```

Comment:
can we please do this in a more extensible way with a map, we're going to have this problem in future for extra assets.